wpimath/src/main/java/edu/wpi/first/math/controller/ElevatorFeedforward.java

```diff
@@ -99,76 +123,150 @@ public double getKv() {
   }
 
   /**
-   * Returns the acceleration gain.
+   * Returns the acceleration gain in V/(m/s²).
    *
    * @return The acceleration gain.
    */
   public double getKa() {
     return ka;
   }
 
+  /**
+   * Returns the period in seconds.
+   *
+   * @return The period in seconds.
+   */
+  public double getDt() {
+    return m_dt;
+  }
+
   /**
    * Calculates the feedforward from the gains and setpoints.
    *
    * @param velocity The velocity setpoint.
    * @param acceleration The acceleration setpoint.
    * @return The computed feedforward.
    */
+  @SuppressWarnings("removal")
+  @Deprecated(forRemoval = true, since = "2025")
   public double calculate(double velocity, double acceleration) {
     return ks * Math.signum(velocity) + kg + kv * velocity + ka * acceleration;
   }
 
   /**
-   * Calculates the feedforward from the gains and setpoints.
+   * Calculates the feedforward from the gains and velocity setpoint (acceleration is assumed to be
+   * zero).
+   *
+   * @param velocity The velocity setpoint.
+   * @return The computed feedforward.
+   */
+  @SuppressWarnings("removal")
+  @Deprecated(forRemoval = true, since = "2025")
+  public double calculate(double velocity) {
+    return calculate(velocity, 0);
+  }
+
+  /**
+   * Calculates the feedforward from the gains and setpoints assuming discrete control when the
+   * setpoint does not change.
+   *
+   * @param setpoint The velocity setpoint.
+   * @return The computed feedforward.
+   */
+  public Voltage calculate(LinearVelocity currentVelocity) {
+    return calculate(setpoint, setpoint);
+  }
+
+  /**
+   * Calculates the feedforward from the gains and setpoints assuming discrete control.
    *
    * <p>Note this method is inaccurate when the velocity crosses 0.
    *
    * @param currentVelocity The current velocity setpoint.
    * @param nextVelocity The next velocity setpoint.
-   * @param dtSeconds Time between velocity setpoints in seconds.
    * @return The computed feedforward.
    */
-  public double calculate(double currentVelocity, double nextVelocity, double dtSeconds) {
-    // Discretize the affine model.
+  public Voltage calculate(LinearVelocity currentVelocity, LinearVelocity nextVelocity) {
+    // For an elevator with the model
+    // dx/dt = −kᵥ/kₐ x + 1/kₐ u - kg/kₐ - kₛ/kₐ sgn(x),
     //
-    //   dx/dt = Ax + Bu + c
-    //   dx/dt = Ax + B(u + B⁺c)
-    //   xₖ₊₁ = eᴬᵀxₖ + A⁻¹(eᴬᵀ - I)B(uₖ + B⁺cₖ)
-    //   xₖ₊₁ = A_d xₖ + B_d (uₖ + B⁺cₖ)
-    //   xₖ₊₁ = A_d xₖ + B_duₖ + B_d B⁺cₖ
+    // where
```

Comment:
Add newlines between text and math blocks, and indent math blocks by two spaces.